### DISCUSSION TITLE
Can I bring custom domain for Directus APIs

### DISCUSSION THREAD

**Discussion by gauravkrp** [directus/directus#17405] [Archive]
Currently directus exposes APIs for its tables as `myappdomain.com/items/table_name` 
I will like to know if it is possible to bring my own domain and expose/handle API routes as `api.mydomain.com/table_name`

**Comment by JonathanSchndr** [comment]
> Currently directus exposes APIs for its tables as `myappdomain.com/items/table_name` I will like to know if it is possible to bring my own domain and expose/handle API routes as `api.mydomain.com/table_name`

Yes of course! You can create your own custom endpoints: https://docs.directus.io/extensions/endpoints.html
